projects/1.20/assets/game-discs/gamediscs/lang/zh_cn.json

```
@@ -0,0 +1,51 @@
+{
+    "creativetab.game_discs_tab": "游戏光碟",
+  
+  
+    "gamediscs.flappy_bird": "飞翔的蜜蜂",
+    "gamediscs.slime": "贪吃史莱姆",
+    "gamediscs.blocktris": "消除方块",

```

```suggestion
    "gamediscs.blocktris": "俄罗斯“方”块",
```
群内讨论的结果之一
应该算是保留了原有游戏的名称，也带点block的意思

---

这个改完就OK了，建议更改成这个